nss/src/group/mod.rs

```diff
@@ -123,17 +123,17 @@ fn get_entry_by_name(name: String) -> Response<Group> {
     })
 }
 
-/// group_entry_to_group converts a GroupEntry to a libnss::Group.
-fn group_entry_to_group(entry: GroupEntry) -> Group {
-    Group {
-        name: entry.name,
-        passwd: entry.passwd,
-        gid: entry.gid,
-        members: entry.members,
+/// group_to_group_entry converts a authd::Group to a libnss::Group.
+fn group_to_group_entry(group: Group) -> NSSGroup {
```

Comment:
The type `GroupEntry` doesn't exist anymore, so we should use a different name here, IMO. Maybe `authd_group_to_group`? So it indicates better what's being converted to what.

Reply:
`entry` is still the term used by NSS, for example in the names of the hooks we define here: `get_all_entries`, `get_entry_by_gid`, `get_entry_by_name`.

Reply:
I renamed it to `authd_group_to_group_entry`